server/conf/settings.py

```diff
@@ -47,5 +47,5 @@
                      'world.content.prototypes_mobs',
                      'world.content.prototypes_weapons'
                      )
-BASE_BATCHPROCESS_PATHS = ['world.content', 'evennia.contrib']
+BASE_BATCHPROCESS_PATHS = ['world.content']
```

Comment:
I don't remember why I kept `evennia.contrib`, probably to keep test scripts in contrib available? But I agree it seems safer to leave it out.